composer.json

```diff
@@ -14,9 +14,16 @@
             "email": "email@domain.com"
         }
     ],
+    "repositories": [
```

Comment:
Might want to publish this to packagist so this isn't needed.

Reply:
I'll discuss with @dlh01